en/modules/modules_system.md

```diff
@@ -945,3 +945,50 @@ work_queue <command> [arguments...]
 
    status        print status info
 ```
+
```

Comment:
@Perrrewi You don't need this doc because it will be auto-generated from the sources when the module goes in.

Reply:
I moved these instructions into the module, thanks for the review!